.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "examples/specs"]
+	path = examples/specs
+	url = git@github.com:Componolit/RecordFlux-specifications.git
```

Comment:
A https URL would be better for people without a GitHub account.

Reply:
Changed in 45c768d6f782bb88ce5a74ec46813a2ac4f02954.